wrappers/javascript/shared/README.md

```diff
@@ -21,6 +23,37 @@ const getSchemaRequest = new GetSchemaRequest({
   schemaId: 'J6nTnUo3YLayzc2GUUctb1:2:MyName:1.0',
 })
 
-const schemaObject =
-  await pool.submitRequest({ requestHandle: getSchemaRequest.handle })
+const schemaResponse = await pool.submitRequest(getSchemaRequest)
+```
+
+## Platform independent setup
+
+If you would like to leverage the Indy VDR libraries for JavaScript in a platform independent way you need to add the `indy-vdr-shared` package to your project. This package exports all public methods.
+
+Before calling any methods you then need to make sure you register the platform specific native bindings. You can do this by importing the platform specific package. You can do this by having separate files that register the package, which allows the React Native bundler to import a differnet package:
+
+```typescript
+// register.ts
+import 'indy-vdr-react-native'
+```
+
+```typescript
+// register.native.ts
+import 'indy-vdr-react-native'
+```
```

Comment:
This confuses me slightly as you import the same package. Is that the point or am I missing something?

Reply:
Good catch. One should be nodejs :)